charts/zigbee2mqtt/templates/statefulset.yaml

```diff
@@ -78,6 +78,40 @@ spec:
             - mountPath: /app/data/
               name: data-volume
           resources: {{- toYaml .Values.statefulset.resources | nindent 12 }}
+      initContainers:
+        - name: yq
+          ## hardcoding here the image. It will be either available in z2m or it will be added to config.
+          ## not creating entries config yet so users don't couple with it, causing migration issues.
+          image: "mikefarah/yq:4.45.1"
```

Comment:
@Koenkk  alpine has yq [package](https://pkgs.alpinelinux.org/package/v3.15/community/x86/yq). aprox 5MB of  binary. I saw your image is very slim, not sure how you feel about adding yq to z2m. It't not a big deal for me to create the config here, it just felt handy since all z2m is based on yaml.

Reply:
Then let's keep it here (since it will only be used for k8s)